tests/func/utilities/test_peirce.py

```diff
@@ -1,12 +1,45 @@
 """Test for Peirce criterion data rejection."""
 
 
+from hypothesis import given, strategies as st
 import pytest
 import numpy as np
 
 from rimseval.utilities import peirce
 
 
+def test_peirce_criterion_ldiv_zero():
+    """Choose a minimum ldiv if it is zero (erfc turns 0 at large values)."""
+    # define values such that it turns out to be zero
+    n_tot = 100
+    n = 1e3  # this brings `ldiv` numerically to zero
```

Comment:
setting `n` to a high value will numerically bring `ldiv` to zero, since `r_new` is generally `<1` and `ldiv=r_new ** n`. However, `ldiv = 0` would lead to issues further down, therefore we set it here such that it cannot turn zero.